.gitignore

```diff
@@ -14,5 +14,6 @@
 /logs/*.log
 /logs/*.log.*
 /media/attachments
+/media/attachment_normalizations
```

Comment:
Keby si si to otestoval, tak by sa Ti objavili necomitnute subory v `/media/attachment_normalizations` z file fieldu.